modules/light-clients/08-wasm/testing/wasm_endpoint.go

```diff
@@ -0,0 +1,42 @@
+package wasmtesting
+
+import (
+	"github.com/stretchr/testify/require"
+
+	types "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+)
+
+type WasmEndpoint struct {
+	*ibctesting.Endpoint
+}
+
+var (
+	CodeHash               = []byte("01234567012345670123456701234567")
+	contractClientState    = []byte{1}
+	contractConsensusState = []byte{2}
+)
+
+// CreateClient creates an wasm client on a mock cometbft chain.
+// The client and consensus states are represented by byte slices
+// and the starting height is 1.
+func (endpoint *WasmEndpoint) CreateClient() (err error) {
```

Comment:
```suggestion
func (endpoint *WasmEndpoint) CreateClient() error {
```

I really don't like named return args 😅 magic variable initialisation scares me. can we refactor this to not use them? I think it will all work as expected if we just removed the name `err`.